### DISCUSSION TITLE
Is there a way to add classes to the flowbite-react components

### DISCUSSION THREAD

**Discussion by wahabshah21** [themesberg/flowbite-react#479] [Q&A] [answered]
Is there a way to add classes to the flowbite-react components? I am using tabs component and I would like to change the style according to my own requirements. Is it possible to do so? 

I have not found anything yet and className doesn't work on Tabs.Item or Tabs.Group. Kindly point me in the right direction whether it is possible or not or if there is a way to handle it.

**Comment by rluders** [comment]
@wahabshah21 you can use the flowbite-react theme support. There are some discussions here about it, and how to use it, as well at the Discord server.

We are currently working at #443 which will give more flexibility, but TBH, the `Tab` and `Accordion` components are the ones that we need to do some work to simplify a few things in order to make it fully customizable.

**Comment by tulup-conner** [comment, marked as answer]
See this discussion: https://github.com/themesberg/flowbite-react/discussions/92

Additionally, components let you pass a `className` to override the default classes now.
